### PR TITLE
Round down source_registration_time to whole day

### PR DESCRIPTION
**Pull request by linnan-github**

### CODE REVIEW

**Comment by linnan-github** [comment]
johnivdel@, could you PTAL, thanks!

**Review by palenica — APPROVED** [review]

File: AGGREGATE.md [hunk 1]
```diff
@@ -226,7 +226,7 @@ utilize techniques like retries to minimize data loss.
   offline devices reporting late).
 
 * The `source_registration_time` will represent (in seconds since the Unix Epoch) the
-  time the source event was registered, rounded to the nearest whole day.
+  time the source event was registered, rounded down to whole day.
```
Comment (alexmturner) [inline AGGREGATE.md:229]: nit: maybe "rounded down to _a_ whole day"

File: AGGREGATE.md [hunk 2]
```diff
@@ -236,9 +236,9 @@ utilize techniques like retries to minimize data loss.
   string therefore must be forwarded to the aggregation service unmodified. The
   reporting origin can parse the string to access the encoded fields.
   
-* The `source_registration_time` within `shared_info` will represent (in seconds 
-  since the Unix Epoch) the time the source event was registered, rounded to the
-  nearest whole day.
+* The `source_registration_time` within `shared_info` will represent (in seconds
+  since the Unix Epoch) the time the source event was registered, rounded down
+  to whole day.
```
Comment (alexmturner) [inline AGGREGATE.md:241]: ditto

**Review by alexmturner — APPROVED** [review]
LGTM, one nit

**Review by johnivdel — APPROVED** [review]
lgtm with comments

File: AGGREGATE.md (hunk 2 above)
Comment (alexmturner) [reply AGGREGATE.md:241]: n/a see above

File: AGGREGATE.md [hunk 3]
```diff
@@ -236,9 +236,9 @@ utilize techniques like retries to minimize data loss.
   string therefore must be forwarded to the aggregation service unmodified. The
   reporting origin can parse the string to access the encoded fields.
   
-* The `source_registration_time` within `shared_info` will represent (in seconds 
-  since the Unix Epoch) the time the source event was registered, rounded to the
-  nearest whole day.
+* The `source_registration_time` within `shared_info` will represent (in seconds
```
[the remark this answers is not in the document]
Comment (alexmturner) [reply AGGREGATE.md:239]: Oh yep, I think this is just a rebase artifact. Agree re: removal

[the remark this answers is not in the document]
Comment (alexmturner) [reply AGGREGATE.md:239]: (Nvm -- not a rebase artifact. I just missed removing this)

**Pull request merged**